azure-pipelines.yml

```diff
@@ -64,8 +64,7 @@ jobs:
     submodules: true
   - script: |
       ./autogen.sh
-      fakeroot dpkg-buildpackage -b -us -uc -Tbinary-syncd-vs
-      cp ../*.deb .
+      fakeroot dpkg-buildpackage -b -us -uc -Tbinary-syncd-vs -j$(nproc) && cp ../*.deb .
```

Comment:
just of out curiosity why this was merged to 1 line?

Reply:
this is to prorogate the error code to the script